src/tools/schema2kotlin.ts

```diff
@@ -25,10 +25,10 @@ const keywords = [
 ];
 
 const typeMap = {
-  'T': {type: 'String',  defaultVal: '""',    decodeFn: 'decodeText()'},
-  'U': {type: 'String',  defaultVal: '""',    decodeFn: 'decodeText()'},
-  'N': {type: 'Double',  defaultVal: '0.0',   decodeFn: 'decodeNum()'},
-  'B': {type: 'Boolean', defaultVal: 'false', decodeFn: 'decodeBool()'},
+  'T': {type: 'String?',  defaultVal: 'null',    decodeFn: 'decodeText()'},
+  'U': {type: 'String?',  defaultVal: 'null',    decodeFn: 'decodeText()'},
+  'N': {type: 'Double?',  defaultVal: 'null',   decodeFn: 'decodeNum()'},
+  'B': {type: 'Boolean?', defaultVal: 'null', decodeFn: 'decodeBool()'},
```

Comment:
You can probably remove the `defaultVal` field altogether, since it's now always null.

Reply:
Good shout. Addressed in dd694c3